test/pkg/test/path_match_httproute.go

```diff
@@ -3,20 +3,21 @@ package test
 import (
 	"github.com/aws/aws-application-networking-k8s/pkg/latticestore"
 	"github.com/samber/lo"
-	v1 "k8s.io/api/core/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
 	"sigs.k8s.io/gateway-api/apis/v1beta1"
 	"strconv"
 )
 
-func (env *Framework) NewPathMatchHttpRoute(parentRefsGateway *v1beta1.Gateway, services []*v1.Service) *v1beta1.HTTPRoute {
+func (env *Framework) NewPathMatchHttpRoute(parentRefsGateway *v1beta1.Gateway, backendRefObjects []client.Object,
+	gwListenerSectionName string) *v1beta1.HTTPRoute {
 	var rules []v1beta1.HTTPRouteRule
```

Comment:
Use a more generic interface `client.Object`, so that backendRefObjects could be either Service or Service Import